Chat: describe Editing feature (#7293)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChat/1 Configuration/editing/allowDeleting.md

---
id: dxChat.Options.editing.allowDeleting
type: Boolean | function(options)
default: false
---
---
##### shortDescription

> Specifies whether a user can delete messages. It is called for each message when defined as a function.

Reviewer: ```suggestion
Specifies whether a user can delete messages. Called for each message when defined as a function.
```

##### param(options): Object
Information about the current message.

##### field(options.component): dxChat
Chat's instance.

##### field(options.message): Message
Message data.

##### return: Boolean
`true` if the message can be deleted; otherwise `false`.

---

> If set to `false`, no messages can be deleted. If set to `true`, only current user can delete their messages.

Reviewer: ```suggestion
If `false`, no messages can be deleted. If `true`, only the current user can delete their messages.
```
